src/bramble/client/main.js

```diff
@@ -309,6 +311,14 @@ define([
                         _state.allowJavaScript = data.allowJavaScript;
                     } else if (eventName === "tutorialVisibilityChange") {
                         _tutorialVisible = data.visible;
+                    } else if (eventName === "TagHintsChange") {
```

Comment:
This is a leaky abstraction, and I'd like to harmonize these event types into a single `autoCompleteChange` event.  See my comment below.